### PR TITLE
Set graph's random seed when training

### PR DESCRIPTION
**Pull request by reuben**

### CODE REVIEW

**Comment by lissyx** [comment]
@reuben Looks like something regressed into `+ '[' -z 'she had a due in greasy wash water year' -o -z 'she had out in greasy wash water all year' ']'`

File: util/flags.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def create_flags():
 
     # Initialization
 
-    tf.app.flags.DEFINE_integer ('random_seed',      4567,        'default random seed that is used to initialize variables')
+    tf.app.flags.DEFINE_integer ('random_seed',      4568,        'default random seed that is used to initialize variables')
```
Comment (lissyx) [inline util/flags.py]: Git commit says "75 epochs" but `tc-train-tests.sh` still uses 105 ?

Comment (reuben) [reply util/flags.py]: That's probably why it wasn't failing so often. 75 is from run-ldc93s1.sh

**Comment by reuben** [comment]
The prod test failures are most likely from #1744

**Pull request merged**